static/src/javascripts-legacy/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -0,0 +1,27 @@
+define([
+    'common/modules/experiments/segment-util',
+    'common/modules/experiments/tests/contributions-epic-brexit',
+    'common/modules/experiments/tests/contributions-epic-always-ask-strategy',
+], function (
+    segmentUtil,
+    brexit,
+    alwaysAsk
+) {
+    /**
+     * acquisition tests in priority order (highest to lowest)
+     */
+    var tests = [alwaysAsk, brexit];
+
+    return {
+        getTest: function() {
+            var eligibleTests = tests.filter(function (test) {
+                var t = new test();
+                var forced = new RegExp('^#ab-' + t.id).test(window.location.hash);
```

Comment:
I think you may as well test for equality here as the regexp doesn't add that much value (as far as I understand)